docs/source/typing_anti_pitch.rst

```diff
@@ -3,10 +3,12 @@
 Reasons to avoid static type checking
 =====================================
 
-In the words of :pep:`484`::
+In the words of :pep:`484`:
 
-    It should also be emphasized that Python will remain a dynamically typed language, and the
-    authors have no desire to ever make type hints mandatory, even by convention.
+.. code-block:: none
+
```

Comment:
```suggestion
```
The correct formatting for block quotes is simply an indented block (https://docutils.sourceforge.io/docs/user/rst/quickref.html#block-quotes).

Looks like this:
<img width="826" alt="Screenshot 2023-11-26 at 4 53 10 PM" src="https://github.com/python/typing/assets/906600/4f733e62-1413-4a96-8c51-14f8b8c182ba">

Reply:
Yeah, I am aware of that. But the original was using a codeblock, so I wanted to preserve this while fixing the scrollbar and syntax highlighting. If you guys want this changed, we can do that.

Reply:
I suppose @hauntsaninja can confirm but I doubt that was intentional. The text is a quote; we should use the appropriate formatting for quotes.